**Our lab is built on three pillars:** methods development, ovarian cancer biology, and translational impact. 

The main focus of our lab is to use machine learning approaches to deepen our understanding of ovarian cancer and improve patient outcomes. Machine learning methods have revolutionized many aspects of our lives, largely due to the vast amount of data available to train the model. To apply machine learning approaches to ovarian cancer, a rare and heterogeneous disease, our lab utilizes and develops new, advanced methods to take full advantage of currently available data. We seek to do this by carefully integrating heterogeneous, multimodal, and multiscale sources of high-throughput sequencing data.

 

Our lab is interested in building upon the following themes:

Integrating measurements from high- and low-resolution technologies at population-scale: Bulk sequencing technologies are well adopted by the community, cheap and easy to obtain, and therefore cover a wider population range. In contrast, high-resolution measurements, such as single-cell or spatial technologies, are difficult and expensive, allowing only a few samples to be sequenced.  This poses a serious problem when studying ovarian cancer since a small number of samples is unlikely to capture the true heterogeneity of the disease. We are interested in developing machine learning methods that integrate small-scale but high-resolution data with population-scale low-resolution bulk data to find clinically relevant and generalizable tumor features. 

 

Ovarian cancer transcriptomic subtype analysis: Transcriptomically derived high-grade serous ovarian cancer subtypes are associated with differences in survival, but it is unclear how subtypes correlate with or differ across tumor compositions or patient populations. We are interested in developing machine learning methods to help identify how subtypes differ across populations and how they are driven by specific cell-type compositions.

 

Interpretable machine learning approaches: High-throughput sequencing data captures many aspects of the sample, including irrelevant biological and technical noise. We are interested in developing machine learning models that directly provide biologically interpretable and robust results. 

 

**Our lab is dedicated to:**

- Open-source software development and code+data reuse

- Creating a welcoming and supportive lab environment

- Interdisciplinary and collaborative partnerships